source/vscode/changelog.md

```diff
@@ -1,5 +1,48 @@
 # QDK Changelog
 
+## v1.23.0
+
+Below are some of the highlights for the 1.23 release of the QDK.
+
+### Full Qiskit 2 support
+
+The `qdk` python package has been updated to support Qiskit 2 circuit submission to the Azure Quantum service. This is done via the `AzureQuantumProvider` class to get a backend object that can run both Qiskit v1 and v2 circuits. This allows for a simpler submission of Qiskit circuits to Azure as compared to the older approach that required manual QIR compilation before submission. The resulting job objects also handle parsing of the Qiskit output format. The pattern will look similar to:
+
+```python
+provider = AzureQuantumProvider(workspace)
+backend = provider.get_backend(target_name)
+job = backend.run(circuit, shots, job_name)
+counts = job.result().get_counts(circuit)
+```
+
+See [the updated Qiskit submission sample notebook](https://github.com/microsoft/qdk/blob/main/samples/python_interop/submit_qiskit_circuit_to_azure.ipynb) for the new supported method of Azure submission with Qiskit 2.
+
+To make sure you get the updated `qdk` package with this support, please use the command `pip install "qdk[azure]" --upgrade`
+
+### Interactive Circuit Diagrams with Source Code Navigation
+
+Circuit diagrams now display clickable source code locations for gates and qubits in VS Code. Click on any operation box to jump directly to where it was called in your Q# or QASM code, or on a qubit label to jump to its declaration site. In Python Jupyter notebooks, source locations can be enabled via `qsharp.circuit(source_locations=True)` to display hover text with code locations.
```

Comment:
Is operation grouping also enabled by default, or is that just in the dev build?

Reply:
Only in dev builds by default for now.